l10n_ch_pain_direct_debit/demo/dd_demo.yml

```diff
@@ -0,0 +1,79 @@
+-
```

Comment:
yaml files are no more supported in v12. Please delete it and if some records are needed in the demo, convert those records inside the dd_demo.xml file.